rero_ils/modules/holdings/jsonschemas/holdings/holding-v0.0.1.json

```diff
@@ -151,6 +154,114 @@
             }
           }
         },
+        "frequency": {
+          "title": "Frequency",
+          "type": "string",
+          "enum": [
+            "rdafr:1001",
+            "rdafr:1002",
+            "rdafr:1003",
+            "rdafr:1004",
+            "rdafr:1005",
+            "rdafr:1006",
+            "rdafr:1007",
+            "rdafr:1008",
+            "rdafr:1009",
+            "rdafr:1010",
+            "rdafr:1011",
+            "rdafr:1012",
+            "rdafr:1013",
+            "rdafr:1014",
+            "rdafr:1015",
+            "rdafr:1016"
+          ],
+          "form": {
+            "options": [
+              {
+                "label": "annual",
+                "value": "rdafr:1013"
+              },
+              {
+                "label": "biennial",
+                "value": "rdafr:1014"
+              },
+              {
+                "label": "bimonthly",
+                "value": "rdafr:1007"
+              },
+              {
+                "label": "biweekly",
+                "value": "rdafr:1003"
+              },
+              {
+                "label": "daily",
+                "value": "rdafr:1001"
+              },
+              {
+                "label": "irregular",
+                "value": "rdafr:1016"
+              },
+              {
+                "label": "monthly",
+                "value": "rdafr:1008"
+              },
+              {
+                "label": "quarterly",
+                "value": "rdafr:1010"
+              },
+              {
+                "label": "semiannual",
+                "value": "rdafr:1012"
+              },
+              {
+                "label": "semimonthly",
+                "value": "rdafr:1009"
+              },
+              {
+                "label": "semiweekly",
+                "value": "rdafr:1005"
+              },
+              {
+                "label": "three times a month",
+                "value": "rdafr:1006"
+              },
+              {
+                "label": "three times a week",
+                "value": "rdafr:1002"
+              },
+              {
+                "label": "three times a year",
+                "value": "rdafr:1011"
+              },
+              {
+                "label": "triennial",
+                "value": "rdafr:1015"
+              },
+              {
+                "label": "weekly",
+                "value": "rdafr:1004"
```

Comment:
I need to check with @pronguen , but I would say that labels have a capitalized initial on the first word, such as "Three times a year". I'll amend the [guidelines](https://github.com/rero/developer-resources/blob/master/guidelines/json-schemas.md#enum) accordingly.